test/unit/api/regular/combine.cpp

```diff
@@ -0,0 +1,53 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+**/
+//==================================================================================================
+#include "test.hpp"
+
+
+TTS_CASE_TPL("combine", eve::test::scalar::all_types)
+<typename T>(tts::type<T>)
+{
+  constexpr std::size_t max_size_bit = 7; // 128 bytes max
```

Comment:
drop a small comment to explain the logic of the test
Also, should we also do that with logical ?

Reply:
should be better now